### ISSUE TITLE
Warning: Compiling on macOS requires special link args in order to compile correctly

### ISSUE THREAD

**Issue by OldhamMade** [mischov/meeseeks_html5ever#34]
FYI, I'm seeing the following warning when building on macOS BigSur with 0.12.1:

```
Compiling on macOS requires special link args in order to compile
correctly.

Rustler is currently working around this issue in the compiler task.
This will be removed in v1.0.0 in favor of a user supplied .cargo/config
file.

To remove this warning, please create /Users/me/project/deps/meeseeks_html5ever/native/meeseeks_html5ever_nif/.cargo/config
with the following content:

      [target.x86_64-apple-darwin]
      rustflags = [
          "-C", "link-arg=-undefined",
          "-C", "link-arg=dynamic_lookup",
      ]

See https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/MachOTopics/1-Articles/executing_files.html
for more details.
```

**Comment by mischov** [comment]
What are your versions of Elixir and Erlang?

**Comment by mischov** [comment]
Ah, it's just a warning (as you said)- I was able to recreate. Thank you for the report.

**Comment by mischov** [comment]
Released as of v0.13.0.
